src/overview/build-deploy.md

```diff
@@ -9,36 +9,36 @@ Every time you push to a live branch (a git branch with an active environment at
 
 ## Always Be Compiling
 
-Interpreted languages like PHP or Node.js may not seem like they have to be compiled, but with modern package management tools like Composer or npm and with the growing use of CSS preprocessors such as Sass, most modern web applications need a "build" step between their source code and their production execution code.  At Platform.sh, we aim to make that easy.  That build step includes the entire application container, from language version to build tools to your code, rebuilt every time.
+Interpreted languages like PHP or Node.js may not seem like they have to be compiled, but with modern package management tools like Composer or npm, as well as the growing use of CSS preprocessors such as Sass, most modern web applications need a "build" step between their source code and their production execution code.  At Platform.sh, we aim to make that easy.  The build step includes the entire application container&mdash;from language version to build tools to your code&mdash;rebuilt every time.
 
-That build process is under your control, and reruns on every Git push.  Every Git push is a validation not only of your code but of your build process.  Whether that's installing packages using Composer or Bundler, compiling TypeScript or Sass, or building your application code in Go or Java, your build process is vetted every time you push.
+That build process is under your control and runs on every Git push.  Every Git push is a validation not only of your code, but of your build process.  Whether that's installing packages using Composer or Bundler, compiling TypeScript or Sass, or building your application code in Go or Java, your build process is vetted every time you push.
```

Comment:
Disagree on the additional comma here.

Reply:
Contrasted coordinate elements should always be separated with a comma.

So it would be "a validation of your code and of your build process," but "a validation not only of your code, but of your build process." The construction "not only ..., but ..." is what makes the coordinate element require a comma.

Reply:
I"ll buy that.  Stet.